language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -175,6 +176,24 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
                 CompletionItem variableCItem = VariableCompletionItemBuilder.build(varSymbol, varSymbol.getName().get(),
                         typeName);
                 completionItems.add(new SymbolCompletionItem(ctx, symbol, variableCItem));
+
+                TypeSymbol rawType = CommonUtil.getRawType(varSymbol.typeDescriptor());
+
+                if (CommonUtil.isSelfClassSymbol(symbol, ctx, ctx.enclosedModuleMember().get())) {
```

Comment:
We need to check for `ctx.enclosedModuleMember().isPresent()` right? It may not be null as of now, but the idea of optional is that it can be null.

Reply:
Yes, will do.